tinygrad/ops.py

```diff
@@ -80,12 +80,9 @@ def stride(self, _): raise NotImplementedError
 # **************** for Interpreted Buffers ****************
 
 class Interpreted:
-  def __init__(self, buffer, fxn_for_op: Dict[Op, Callable], from_lazybuffer=lambda x: x.realized, to_underlying=lambda x: x._buf, from_underlying=None):
-    self.buffer = buffer
-    self.fxn_for_op = fxn_for_op
-    self.from_lazybuffer = from_lazybuffer
+  def __init__(self, buffer, fxn_for_op: Dict[Op, Callable], from_lazybuffer=lambda x: x.realized, to_underlying=lambda x: x._buf, from_underlying=None, cast_underlying=lambda x,_:x):
```

Comment:
Do you need `cast_underlying` or can you just use UnaryOps.CAST?

Reply:
done